docs/source/processes.rst

```diff
@@ -424,79 +428,185 @@ This section will first describe the basics of this request format, and after go
+accessible with distinct URL afterwards. If the ``outputs`` section is specified, but that one of the outputs defined
+in the :ref:`Process Description <proc_op_describe>` is not specified, that output should be omitted from the produced
+results. For the time being, because only ``reference`` representation is offered for produced output files, this
+filtering is not implemented as it offers no additional advantage for files accessed directly with their distinct URLs.
+This could be added later if ``Multipart`` raw data representation is required.
+Please |submit-issue|_ to request this feature if it is relevant for your use-cases.
 
-.. note::
-    Since most of the time, returned files are not human readable or are simply too large to be displayed, the
-    ``transmissionMode: value`` is rarely employed. Also, it is to be noted that outputs representing ``LiteralData``
-    (which is even more uncommon) would automatically be represented as ``value`` without explicitly requesting it,
-    as there would not be any file to return. If this poses problem or you encounter a valid use-case where ``value``
-    would be useful for your needs, please |submit-issue|_ to request the feature.
+.. fixme::
+    Filtering of ``outputs`` not implemented (everything always available).
+    https://github.com/crim-ca/weaver/issues/380
 
-.. |exec-api| replace:: OpenAPI Execute
-.. _exec-api: `exec-req`_
+Other parameters presented in the above examples, namely ``mode`` and ``response`` are further detailed in
+the following :ref:`proc_exec_mode` section.
+
+.. _proc_exec_mode:
+
+Execution Mode
+~~~~~~~~~~~~~~~~~~~~~
+
+In order to select how to execute a :term:`Process`, either `synchronously` or `asynchronously`, the ``Prefer`` header
+should be specified. If omitted, `Weaver` defaults to `asynchronous` execution. To execute `asynchronously` explicitly,
+``Prefer: respond-async`` should be used. Otherwise, the `synchronous` execution can be requested
+with ``Prefer: wait=X`` where ``X`` is the duration in seconds to wait for a response. If no worker becomes available
+within that time, or if this value is greater than ``weaver.exec_sync_max_wait``, the :term:`Job` will resume
+`asynchronously` and the response will be returned. Furthermore, `synchronous` and `asynchronous` execution of
+a :term:`Process` can only be requested for corresponding ``jobControlOptions`` it reports as supported in
+its :ref:`Process Description <proc_op_describe>`. It is important to provide the ``jobControlOptions`` parameter with
+applicable modes when :ref:`Deploying a Process <proc_op_deploy>` to allow it to run as desired. By default, `Weaver`
+will assume that deployed processes are only `asynchronous` to handle longer operations.
+
+.. versionchanged::
+    By default, every :ref:`proc_builtin` :term:`Process` can accept both modes.
+    All previously deployed processes will only allow `asynchronous` execution, as only this one was supported.
```

Comment:
It's not that it "cannot" but rather that to respect OGC definitions, because `execute-async` only is defined in their process description, it "must not" run them this way. So yes, redeploying them will allow them to run in sync.